be/src/exprs/vectorized/runtime_filter.h

```diff
@@ -254,7 +254,11 @@ class RuntimeBloomFilter final : public JoinRuntimeFilter {
     RuntimeBloomFilter() = default;
     ~RuntimeBloomFilter() override = default;
 
-    RuntimeBloomFilter* create_empty(ObjectPool* pool) override { return pool->add(new RuntimeBloomFilter()); };
+    RuntimeBloomFilter* create_empty(ObjectPool* pool) override {
+        auto* p = pool->add(new RuntimeBloomFilter());
+        p->init(0);
```

Comment:
I think we don't need to init zero.

Reply:
it's because in `init`, we have to initialize min/max

```
    void init(size_t hash_table_size) override {
        init_bloom_filter(hash_table_size);
        init_min_max();
    }
```

Reply:
It's critical when merging runtime filters. think this case:

in global runtime filter, we create a (int) empty runtime filter first `p`, min/max value is not initialized, so min/max value is 0

and when we merge runtime filters like
- a [10, 20]
- b [5, 12]
then we have min/max value like [0, 20]. But right range should be [5,20]

Reply:
if so, I think we should move the init_min_max to constructor， then 1. the _has_min_max is always true, and we needn't _has_min_max field and we needn't check has_min_max also.

Reply:
OK.

Reply:
For some types, it's very hard to construct min/max value.